tests/acceptance/features/bootstrap/WebUIAdminStorageSettingsContext.php

```diff
@@ -84,6 +83,43 @@ public function theAdministratorEnablesTheExternalStorageUsingTheWebui() {
 		);
 	}
 
+	/**
+	 * @Given the administrator has enabled the external storage
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theAdministratorHasEnabledTheExternalStorageUsingTheWebui() {
```

Comment:
```suggestion
	public function theAdministratorHasEnabledTheExternalStorage() {
```
and this code could be moved to one of the contexts for OCC commands.
Then it could also be used by other API or CLI tests if needed.

Reply:
OK,,done!